src/Lower.cpp

```diff
@@ -69,7 +69,8 @@ using std::vector;
 using std::map;
 
 Stmt lower(const vector<Function> &output_funcs, const string &pipeline_name,
-           const Target &t, const vector<IRMutator *> &custom_passes) {
+           const Target &t, std::insert_iterator<std::vector<Module>> submodules,
```

Comment:
Same comment as above, consider just passing a reference to a vector of modules.